hash/msx2_flop.xml

```diff
@@ -7002,6 +7291,19 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="gogo1148" supported="partial">
+		<description>Gogo 11-ji 48-bu Version 4 (Japan)</description>
+		<year>1992</year>
+		<publisher>Fantasy Creators</publisher>
+		<notes>Slight graphics glitch at top line of screem when starting a game.</notes>
+		<info name="alt_title" value="午後１１時４８分 Version 4"/>
```

Comment:
Shouldn’t this be read “Gogo 11-ji 48-pun”?

Reply:
Yes, pun is better. Google translate was wrong once again :(